preload.js

```diff
@@ -34,23 +35,26 @@ class SafeIpcRenderer {
           throw new Error(`Blocked access to unknown channel
             ${channel} ${validChannel} from the renderer`);
         }
+
         return fn.apply(ipcRenderer, [channel].concat(args));
       };
     };
-    this.on = protect(ipcRenderer.on);
-    this.once = protect(ipcRenderer.once);
-    this.removeListener = protect(ipcRenderer.removeListener);
+
+    this.on                 = protect(ipcRenderer.on);
+    this.once               = protect(ipcRenderer.once);
+    this.send               = protect(ipcRenderer.send);
+    this.sendSync           = protect(ipcRenderer.sendSync);
+    this.sendToHost         = protect(ipcRenderer.sendToHost);
+    this.removeListener     = protect(ipcRenderer.removeListener);
     this.removeAllListeners = protect(ipcRenderer.removeAllListeners);
-    this.send = protect(ipcRenderer.send);
-    this.sendSync = protect(ipcRenderer.sendSync);
-    this.sendToHost = protect(ipcRenderer.sendToHost);
   }
 }
 
 window.ipc = new SafeIpcRenderer([
+  'multiwallet',
   'rpc-channel',
```

Comment:
We should prefix this with frontend or backend so we know where the channel is registered when we call it